vulnerabilities/importer.py

```diff
@@ -261,7 +261,9 @@ def from_dict(cls, advisory_data):
                 AffectedPackage.from_dict(pkg) for pkg in advisory_data["affected_packages"]
             ],
             "references": [Reference.from_dict(ref) for ref in advisory_data["references"]],
-            "date_published": date_published.isoformat() if date_published else None,
+            "date_published": datetime.datetime.fromisoformat(date_published)
+            if date_published
+            else None,
```

Comment:
You likely want this done above in plain code rather than inside the dict creation, just before line 257

Reply:
Can you also add a unit test just for this?